packages/Babylonian-UI.package/BPBrowser.class/instance/morphForSidePane.st

```diff
@@ -0,0 +1,5 @@
+what to show
+morphForSidePane
+
+	self sidePaneContent = #none ifTrue: [^ Morph new hide].
```

Comment:
Why do we need the hidden Morph?

Reply:
That line got removed because it is not needed